src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```diff
@@ -2050,14 +2054,14 @@ private PrinterHelper getPrinterHelper() {
 	/**
 	 * @param preprocessors list of {@link CompilationUnitValidator}, which have to be used to validate and fix model before it's printing
 	 */
-	public void setPreprocessors(List<Processor<CtCompilationUnit>> preprocessors) {
-		this.preprocessors = preprocessors;
+	public void setPreprocessors(List<Processor<CtElement>> preprocessors) {
+		this.preprocessors.addAll(preprocessors);
 	}
```

Comment:
`setPreprocessors` add preprocessors, isn't that a bit missleading? Can we rename the method addPreprocessors? (Even if it changes the API?